packages/blocks/src/surface-block/element-model/brush.ts

```diff
@@ -56,6 +58,9 @@ export class BrushElementModel extends ElementModel<BrushProps> {
   @yfield()
   points: number[][] = [];
 
+  @yfield()
+  pressures: number[] = [];
```

Comment:
For existing data without this field, we need to make sure it doesn't break - you can edit in try-blocksuite.vercel.app and export the snapshot to test it.

Just to double check with @doouding that I think this yfield is supposed to store using plain array in yjs, not YArray?

Reply:
> For existing data without this field, we need to make sure it doesn't break - you can edit in try-blocksuite.vercel.app and export the snapshot to test it.
> 
> Just to double check with @doouding that I think this yfield is supposed to store using plain array in yjs, not YArray?

Yes， this is going to be plain array. 

I wonder if the pressure could be store in the points array, make it a [number, number, number] array.

Reply:
It is okay to store it in a points array, but I don't know whether this is a breaking change. I will test this later.

Reply:
Currently, pressure and coordinates are stored in the `points` array, and it is compatible with previous snapshots.